truss/httperr.go

```diff
@@ -0,0 +1,28 @@
+package truss
+
+import "net/http"
+
+// HTTPError encodes information in an error type for the
+// github.com/go-kit/kit/tranport/http.DefaultErrorEncoder to create a
+// meaningful http response
+func HTTPError(err error, statusCode int, headers http.Header) error {
```

Comment:
It is now in `transport_http.go` and is unexported. As promoting importing a service inside a lib seems undesired. If a lib wants to use the `StatusCoder` and `Headerer` functionality then they can copy and paste this into their implementation.